src/lms-harmonizer/.flake8

```
@@ -0,0 +1,11 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+
+[flake8]
```

```suggestion
[flake8]
```

The extract stuff at the top actually breaks this file, causing flake8 to be overly aggressive.

---

✔️